modules/server_processes_manager/README.md

```diff
@@ -0,0 +1,65 @@
+# Server Processes Manager
+
+## Purpose
+
+The server processes manager module is used to get information about processes that were 
+launched asynchronously by other LORIS modules
+
+## Intended Users
+
+The only user that should have access to this module is the person in charge of
+the administration of the LORIS website
```

Comment:
what if there's multiple people in charge? And who's "the person in charge"? The study coordinator or the system administrator?

Reply:
(also missing a period)

Reply:
should replace Loris website by Loris application (to remove possible confusion with Loris.ca)

Reply:
Done